hw2/fastq_filtrator.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct  4 20:41:26 2022
+
+@author: anja1
+"""
+# Импорт модулей для работы путей к файлам
+import os
+import glob
+
+# Функция для подсчета GC
+def GC_count(line):
+    count = 0
+    for i in line.lower():
+        if i == 'g' or i == 'c':
+            count += 1  
+    return round(count/len(line)*100,1)
+
+# Функция для подсчета среднего качества
+def mean_ql(line):
+    q_num =[]
+    for i in line:
+        q_num.append(ord(i)-33)
+    return(round(sum(q_num)/len(q_num),1))
+
+# Фильтр для GC
+def filter_GC(line, start, stop):
+    if GC_count(line) >= start and GC_count(line) <= stop:
+        return True
+    else:
+        return False
```

Comment:
Даже ещё больше упростить
```suggestion
    return start <= GC_count(line) <= stop:
```